packages/language-support/src/antlr-grammar/Cypher25Parser.g4

```diff
@@ -14,20 +14,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-parser grammar Cypher5Parser;
```

Comment:
FYI, the grammar files are uploaded as a zip file somewhere when we build the mono repo if that is helpful.